tests/components/normal-components/switch.test.tsx

```diff
@@ -0,0 +1,18 @@
+import { it, expect } from "bun:test"
+import { getTestFixture } from "tests/fixtures/get-test-fixture"
+
+it("should render an switch", async () => {
+  const { circuit } = getTestFixture()
+  circuit.add(
+    <board width="10mm" height="10mm">
+      <switch name="SW1" type="spst" schX={2} schY={2} />
+      <switch name="SW2" spst schX={-2} schY={-2} />
+      <switch name="SW3" type="spdt" schX={0} schY={0} />
+      <switch name="SW4" schX={2} schY={-2} />
+      <switch name="SW5" isNormallyClosed dpdt schX={-2} schY={2} />
```

Comment:
isNormallyClosed means the switch symbol should be closed, which doesn't work right now (it should be fixed), but i think we should do it in a future PR (feel free to create an issue)